slack-server-integration-common/src/main/java/com/atlassian/plugins/slack/api/client/DefaultSlackClientProvider.java

```diff
@@ -134,6 +142,17 @@ private Authenticator createProxyAuthenticator() {
                     ? Credentials.basic(httpsProxyUser, httpsProxyPassword)
                     : null;
 
+            if (log.isDebugEnabled()) {
```

Comment:
Added logging when client is initialized showing information about which parameters were used.